gulpfile.js

```diff
@@ -197,6 +203,37 @@ async function addDistToIndex() {
 
 const cliVersion = '1.8.5';
 
+async function nugetInstallPortalPackages() {
+    const downloadDir = path.resolve(outdir, 'temp_portal_package', 'download');
+    const unzpiDir = path.resolve(outdir, 'temp_portal_package', 'unzipped_package');
+    const packageName = "CDSStarterPortal"
+    const packageNameToImport = `Adxstudio.${packageName}`
+    const portalPackageOutDir = path.resolve(outdir , 'portal_package')
+
+    await nugetInstall('PowerPortalPackages', 'cdsstarterportal', '9.2.2103.21', downloadDir );
+
+    log.info(`Extracting package zip into folder: ${unzpiDir}`);
```

Comment:
there is no need to also unzip the starterportal.zip that's inside the nupkg: see 'pac deploy package' help or the action.yml, pac will accept the zip with the package dll as well:
> pac package deploy
Usage: pac package deploy [--logFile] [--logConsole] --package
  --package                   path to a package dll or zip file with a package (alias: -p)

Reply:
ahh, I get it: the starterportal nukpg has a zip with the zip....